### DISCUSSION TITLE
Welcome to Life123's Blog / Discussions!

### DISCUSSION THREAD

**Discussion by BrainAnnex** [BrainAnnex/life123#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to blog and connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

To get started, comment below with an introduction of yourself and tell us about your interests.

**Comment by mihaitodor** [comment]
Hey, Mihai here. Software Engineer by training, currently focused on cloud computing, scalability, monitoring and data streaming. I have a degree in Automatic Control and I really enjoyed studying digital signal processing and various linear algebra / numerical methods courses. Since then, I spent the past 15 years working on various software products in multiple corporations, ranging from desktop apps to distributed Cloud-based SaaS (Software as a Service) product backends and cloud infrastructure. My programming language of choice is Golang and I spent quite a bit of time using C++, some Python, etc. I love contributing to Open Source projects and I’m currently involved in a data streaming project called [Benthos](https://benthos.dev/).

I am planning to transition to computational biology in the next few years (thinking to find a PhD somewhere) so I can work on topics such as immunotherapy for treating cancer and / or epigenetic clocks to better understand the ageing process and cellular reprogramming.
